recipes/hometheater.rb

```diff
@@ -17,23 +17,3 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # 
-
-apt_repository "plexapp" do
-  uri "http://plexapp.com/repo"
-  distribution "lucid"
-  components ["main"]
-  action :add
-end
-
-package "plex-archive-keyring" do
-  options "--force-yes"
-  action :install
-  notifies :run, "execute[apt-get update]", :immediately
-end
-
-package "plexmediaserver"
-
-service "plexmediaserver" do
-  provider Chef::Provider::Service::Upstart
-  action [:enable, :start]
-end
```

Comment:
Why remove this recipe? This is what I use, and I'm not about to break my own plex server management.